src/render/react/components/Image/comp.js

```diff
@@ -8,10 +8,8 @@ import {
   styleGetterProp,
 } from "../config";
 
-const fs = require("fs");
-const path = require("path");
-
-const bridge = globalThis.SJSJSBridge;
+const path = globalThis[Symbol.for('tjs.internal.modules.path')];
```

Comment:
ugh, this is kind of ugly. the `fs` import was cleaner before. is there a way to make this a bit more idiomatic maybe?